install-devkit.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -e
+
+# DevKit version
+DEVKIT_VERSION="v0.0.9"
```

Comment:
can this use the root VERSION file to get the latest devkit version ? This way we won't have to update it on every release

Reply:
```suggestion
DEVKIT_VERSION=$(curl -fsSL https://raw.githubusercontent.com/Layr-Labs/devkit-cli/main/VERSION)
```